components/NavBar/NavBar.jsx

```diff
@@ -4,6 +4,8 @@ import Link from "next/link";
 
 import DrawerToggleButton from "components/SideDrawer/DrawerToggleButton";
 
+import Notice from "components/Notice/Notice";
```

Comment:
Looks great! For now since we aren't using this lets not import. Other than that all's good!